kotsadm/api/src/snapshots/resolvers/veleroClient.ts

```diff
@@ -66,7 +66,7 @@ export class VeleroClient {
     if (!cluster) {
       throw new Error("No cluster available from kubeconfig");
     }
-    this.server = cluster.server;
+    this.server = "https://kubernetes.default"
```

Comment:
This was the cluster IP of the kubernetes service. No proxy includes the whole service CIDR but this node library does not handle cidrs.